### DISCUSSION TITLE
Set Up with Cellphone

### DISCUSSION THREAD

**Discussion by Jafranta** [SmoothWAN/SmoothWAN#42] [General & Help]
I didn't have a stable internet connection the first time I went to set up SmoothWAN. I am attempting to set it up a second time with an ethernet connection. I can get to the part where I choose "SmoothWAN" on Wi-Fi settings. I now am trying to go to the webpage http://172.17.17.2/ and I am unable to get anything to load. I am getting an error message stating that it could not connect to the server. Any help is appreciated!

Thanks

**Comment by TalalMash** [comment]
Hi! Can you access the webpage from ethernet instead of Wi-Fi?

**Reply by Jafranta** [reply]
I'm trying a second time with ethernet plugged in and again no luck. I am getting an Aternos 9271 wi-fi dongle tomorrow (hopefully no issues). I tried tethering with an iPhone. It will show that it is picking up the connection (command line). Is there a way I can get back into the OS screen to check to see if it is tethered? Also, I'm new to the command line and Raspberry Pi. Is there a way I can access the wifi setting to have it access it over my home wi-fi?

I tried accessing the website from Chrome and Safari on my iPhone. I also tried accessing SmoothWAN through a computer by connecting to a web browser. I can get to the initial log-in screen for the wifi setting and enter the password. After then, I get the same message of the website cannot be displayed.  

Thanks again for the help. I'm really excited to get this to work and help with streaming issues.

**Reply by TalalMash** [reply]
So, you connected a laptop/desktop to the Pi via ethernet, and the webpage didn't load?

**Comment by Jafranta** [comment]
1.) I tried connecting the Raspberry Pi through Ethernet cord from cable modem. I tried loading webpage from cell phone and another computer another computer web browser that was connected to SmoothWAN via wi-fi connection (like instructions show for cell phone) but just using a computer. 

2.) I tried tethering the Raspberry Pi via usb cord to an iPhone. Again, I tried checking the web browser on the iPhone or another computer web browser that was connected to SmoothWAN via the Wi-Fi connection. 

3.) I repeated the above step but switched web browsers on my cellphone from Safari to Google Chrome. Again, no luck. 

4.)  I tried connecting the set up the way I had it the evening before, with the Ras Pi tethered with a usb cord to a Verizon MiFi hotspot. I could not duplicate what I had got to the previous night. 

I could not access the webpage through all three setups. I know the Raspberry Pi/SmoothWAN is broadcasting via wi-fi because it picks up on the other devices when I try connecting to SmoothWAN.

**Reply by TalalMash** [reply]
Since you mentioned you have lossy internet, try redownloading the image and reflash, it's probably corrupted, you should be able to access the webpage in isolated setup, do not connect any internet source or device yet:
Laptop ethernet ---- Pi's Ethernet

You can use https://emn178.github.io/online-tools/sha256_checksum.html to check the file for errors, latest release checksum:
`4a681fdc8690756317bf26c78488af84f549e2a0b6ecff1ae31dbba5a3efa7df` for `smoothwan-0.99.6-hotfix1-bcm27xx-bcm2711-rpi-4-ext4-factory.img.gz`

**Reply by Jafranta** [reply]
I was able to reflash the SD card and reload it on the Pi with no internet. I was able to get to the screen under Speedify/VPN  where you hit "Trigger/Install Update". Since I did not have any connected internet, it did not download the update. Should I try again with the internet or is there something else you want me to check for? Thanks for the help.

**Reply by TalalMash** [reply]
I see, you may have misinterpreted the ethernet port for the Pi, its only for providing internet and a LAN port, to add internet sources use the USB ports with adapter or tether, check the Wiki regarding iOS tethering, it's crippled in the last few versions and it may not work.
